Remove "work in progress" banners and include as text within pages - to make it less prominent (#325)
EVERSE-ResearchSoftware/RSQKit: index.md

Research software roles (e.g. Research Software Engineers (RSEs) and Researchers who code) are included to document the sources and practitioners of research software. This brings attention to the need for credit and career paths for research software related roles.

> RSQKit is currently developed as part of the [EVERSE](https://everse.software) project, with external engagement with the wider community through the open [EVERSE Network](https://everse.software/network/). The current version of RSQKit is still work in progress - any content should not be considered final at this stage. If you find something missing or want to comment, please get in touch with the [RSQKit Editoral Board](https://everse.software/RSQKit/editorial_board).

Author: ```suggestion
RSQKit is currently developed as part of the [EVERSE](https://everse.software) project, with external engagement with the wider community through the open [EVERSE Network](https://everse.software/network/). The current version of RSQKit is still work in progress - if you find something missing, incorrect or want to comment, please get in touch with the [RSQKit Editoral Board](https://everse.software/RSQKit/editorial_board).
```
